### PR TITLE
Jberet-511 Replace Batch Applications for the Java Platform references in java doc with Jakarta Batch

### PR DESCRIPTION
**Pull request by parsharma**
Issue: https://issues.redhat.com/browse/JBERET-511

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
Hello, @parsharma in your changes for the README file you also need to change the Jakarta Batch links. In another issue https://github.com/jberet/jsr352/pull/162 I have covered it. If you don't mind you can undo README file changes otherwise, it'll create unnecessary conflicts.

**Comment by parsharma** [comment]
Sure, I have resolved the conflicts. Thanks @RanabirChakraborty

File: README.md [hunk 1]
```diff
@@ -51,6 +52,7 @@ Increase `ulimit` to avoid such errors. For example,
 * [JBeret-dev Mailing List](https://lists.jboss.org/mailman/listinfo/jberet-dev)
 * [WildFly Forum](https://groups.google.com/g/wildfly)
 * [JBeret Forum](https://developer.jboss.org/en/jberet/)
+* [Jakarta Batch Expert Group Discussion](https://java.net/projects/jbatch/lists/public/archive)
```
Comment (chengfang) [inline README.md:55]: I think we should refer users to the latest location: https://github.com/eclipse-ee4j/batch-api/issues

Comment (parsharma) [reply README.md]: Changes done.

**Pull request merged**